### PR TITLE
User can assign reward to request

### PR DESCRIPTION
**Pull request by jysmys**
### User can assign a reward to a request
[PT link](https://www.pivotaltracker.com/story/show/173332417)

```
As a user
in order to reward the helper
I would like to set a reward amount to the request
```

- Add permitted parameter :reward to create method
- Add db column  to quest model

**To be done, after Update quest index action to include the reward (serializer)**

### CODE REVIEW

**Review by erikbjoern — APPROVED** [review]
Looks good! Nice error handling.

**Review by designerofthing — APPROVED** [review]

**Review by PaulineBA — APPROVED** [review]

**Review by ViaMarcus — APPROVED** [review]

File: app/controllers/api/requests_controller.rb [hunk 1]
```diff
@@ -21,8 +19,20 @@ def index
 
   private
 
+  def render_error_message(errors)
+    if errors.full_messages.one?
+      error_message = errors.full_messages.first
```
Comment (tochman) [inline app/controllers/api/requests_controller.rb:24]: Why not just call `.to_sentence` on `errors.full_messages`?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+  private
+
+  def check_for_unpermitted_params
+    ActionController::Parameters.action_on_unpermitted_parameters = :raise
+  end
+
+  def unpermitted_parameters(e)
+    render json: { message: e.message }, status: 422
+  end
```
Comment (tochman) [inline app/controllers/application_controller.rb:16]: I think I'm missing something. Is there a request spec for this functionality. What exactly are you achieving with this?

Comment (jysmys) [reply app/controllers/application_controller.rb:16]: no not needed, corpse ....

File: app/controllers/api/requests_controller.rb (hunk 1 above)
Comment (jysmys) [reply app/controllers/api/requests_controller.rb:24]: bad knowledge from my side.... I will try it out!

**Pull request merged**